src/Umbraco.Web.UI.Client/src/init.js

```diff
@@ -134,31 +133,31 @@ app.run(['$rootScope', '$route', '$location', '$cookies', 'urlHelper', 'appState
         //global state query strings without force re-loading views.
         //We can then detect if it's a location change that should force a route or not programatically.
         $rootScope.$on('$routeUpdate', function (event, next) {
-            if (!currentRouteParams) {
+            if (!currentRouteParams || !currentRouteParams.params) {
                 //if there is no current route then always route which is done with reload
                 $route.reload();
             } else {
-                var toRetain = navigationService.retainQueryStrings(currentRouteParams, next.params);
+                var toRetain = navigationService.retainQueryStrings(currentRouteParams.params, next.params);
                 //if toRetain is not null it means that there are missing query strings and we need to update the current params.
                 if (toRetain) {
                     $route.updateParams(toRetain);
                 }
                 //check if the location being changed is only due to global/state query strings which means the location change
                 //isn't actually going to cause a route change.
-                if (navigationService.isRouteChangingNavigation(currentRouteParams, next.params)) {
+                if (navigationService.isRouteChangingNavigation(currentRouteParams.pathParams, next.pathParams)) {
                     //The location change will cause a route change, continue the route if the query strings haven't been updated.
                     $route.reload();
```

Comment:
Now this if statement isn't true, when changing clicking the different content nodes, so it doesn't reload the view.

Reply:
@nul800sebastiaan can you confirm this issue? It seems the changes cause it no longer call `$route.reload();` here.